### PR TITLE
Api With .Net core and Mongo Db

### PR DESCRIPTION
**Pull request by ayushyadav2001**
I Have created the Api using  Asp.Net core and Mongo db and Implemented all the CRUD Operations and Tested using Postman. Please Checkout this.
Thank You
Ayush YADAV

### CODE REVIEW

File: Controllers/TournamentController.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        {
+            MongoClient dbClient = new MongoClient(_configuration.GetConnectionString("TournamentAppCon"));
+
+            var dbList = dbClient.GetDatabase("Tournament").GetCollection<Tournament>("TournamentDetails").AsQueryable();
+
+            return new JsonResult(dbList);
+        }
+        [HttpPost]
+
+        public JsonResult Post([FromBody] Tournament tou)
```
Comment (mukundkatpatal) [inline Controllers/TournamentController.cs:33]: provide a better variableName

Comment (mukundkatpatal) [reply Controllers/TournamentController.cs:33]: why is there no validation in place?

File: Controllers/TournamentController.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        }
+
+
+        [HttpPut]
+        public JsonResult Put([FromBody] Tournament tou)
+        {
+
+            MongoClient dbClient = new MongoClient(_configuration.GetConnectionString("TournamentAppCon"));
+
+            var filter = Builders<Tournament>.Filter.Eq("TournamentId", tou.TournamentId);
```
Comment (mukundkatpatal) [inline Controllers/TournamentController.cs:52]: what if the filter does not return a record for that id?

File: Models/Tournament.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        public ObjectId Id { get; set; }
+
+        public int TournamentId { get; set; }
+
+        public string TournamentName { get; set; }
+
+        public string Place { get; set; }
+
+        public string StartDate { get; set; }
```
Comment (mukundkatpatal) [inline Models/Tournament.cs:18]: how can this be a string?

File: Models/Tournament.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
+﻿using MongoDB.Bson;
+using System;
+using System.Collections.Generic;
```
Comment (mukundkatpatal) [inline Models/Tournament.cs:3]: remove unused imports

File: Models/Tournament.cs (hunk 3 above)
Comment (ayushyadav2001) [reply Models/Tournament.cs:18]: sir I will change it to DateTime

File: Models/Tournament.cs (hunk 4 above)
Comment (ayushyadav2001) [reply Models/Tournament.cs:3]: Sure sir I will

**Pull request closed without merging**